README.md

```diff
@@ -8,49 +8,231 @@ Table of Contents
+2. **Glue Job:** This mode applies when there are already flagged sales in the system. It's an automated scheduled job that flags new, unflagged sales.
+3. **Manual Update:** This mode is used when sales need to be re-flagged, either due to errors or methodology updates. This allows for the selective re-flagging of sales.
+
+### Initial Run Mode
+
+This is the starting point for the Sales Val Pipeline when no sales are flagged.
 
 ```mermaid
 graph TD
+    A{{"No sales are flagged"}}
+    B[Run initial_flagging.py]
+    C[Flag sales as outliers or non-outliers<br>with Version = 1]
+    D[Join flags to<br>default.vw_pin_sale]
+    E[Save results to S3 with unique run ID]
+
+    A -->|Initial setup| B
+    B -->|Flag sales| C
+    C -->|Store flags| D
+    D -->|Persist results| E
+```
 
-    A{{No sales are flagged}}
-    B[Run initial_flagging.py locally]
+### Glue Job Mode
 
-    C[Flags added to sales<br>via flagging.py with<br>Version = 1]
-    D[Flags joined to<br>default.vw_pin_sale]
+This automated job runs on a schedule to process unflagged sales when there are already flagged sales present.
 
-    A --> B
-    B -- Sales pulled from within<br>specified time window --> C
-    C -- Results saved to S3<br>with unique run ID --> D
+```mermaid
+graph TD
+    A[Schedule triggers glue job]
+    B{{"Some sales are already flagged"}}
+    C[Ingest data for unflagged sales]
+    D[Run flagging model within glue job]
+    E[Write sales data to sale.flag]
+    F[Join flags to<br>default.vw_pin_sale]
+    G[Save results to S3 with unique run ID]
+
+    A -->|Trigger| B
+    B -->|Process new sales| C
+    C -->|Run model| D
+    D -->|Output flags| E
+    E -->|Join data| F
+    F -->|Persist results| G
+```
 
-    E{{Some sales need re-flagging}}
-    F[Subset sales in yaml, run<br>manual_update.py locally]
-    G[If sale already flagged<br>increment Version += 1]
-    H{If sale unflagged,<br>assign Version = 1}
-    I[Flags update existing<br>default.vw_pin_sale records]
+### Manual Update Mode
 
-    E --> F
-    F --> G
-    F --> H
-    H --> G
-    G -- Results saved to S3<br>with new run ID --> I
+This mode allows for the manual re-flagging of sales due to errors or when an update in the methodology is required.
 
+```mermaid
+graph TD
+    A{{"Sales must be re-flagged"}}
+    B[Specify subset in yaml]
+    C[Run manual_update.py]
+    D[Increment version if sale already flagged]
+    E[Assign Version = 1 if sale unflagged]
+    F[Update flags in default.vw_pin_sale]
+    G[Save results to S3 with new run ID]
+
+    A -->|Manual selection| B
+    B -->|Run update| C
+    C -->|Version check| D
+    D -->|Update process| F
+    C -->|New flag| E
+    E -->|Update process| F
+    F -->|Persist results| G
 ```
 
+
+## Flags at a Glance
+
+Sales from 2014 - present have been processed using our sales validation program. We flag properties from residential and condo data with the following specifications:
```

Comment:
**suggestion:** I would add a note here about what you mean by "exclusions".

Reply:
I added some context here but I could use your help with it. Not sure exactly what to add. @dfsnow